android/res/values/styles.xml

```diff
@@ -193,10 +193,10 @@
     </style>
 
     <!-- Menu Media Type end -->
-
+    <!--v21-->
```

Comment:
Q: is this some sort of system annotation that android will use? or just documentation?